docs/WithdrawingPaymentsFromFeeds.md

```diff
@@ -0,0 +1,12 @@
+# Withdrawing Payments from Feeds
+
+When Node Operators report data to a feed, they are compensated with LINK thats owned by the feed. Node operators can then withdraw the funds owed to them via gauntlet using this section below
+
+## Sample Command Usage
+
+Make sure to set up gauntlet using the steps described in [Getting Started With Gauntlet](./GettingStartedWithGauntlet.md) before attempting to run the following command
```

Comment:
Can you add an example on how to configure the `/networks/.env.NETWORK` env file inside `GettingStartedWithGauntlet.md`?
You can add the mainnet case. For gauntlet to run we just need a valid `NODE_URL`, and we can suggest some of the Terra public endpoints (https://docs.terra.money/docs/develop/endpoints.html#public-rpc)

Reply:
sounds good, but I think the network files require the lcd endpoints from here: https://docs.terra.money/docs/develop/endpoints.html#public-lcd instead of the rpc endpoints you linked to

Reply:
Right! Should be the LCD